assets/section-email-signup-banner.css

```diff
@@ -1,3 +1,36 @@
+.email-signup-banner .newsletter-form,
```

Comment:
@melissaperreault @wiktoriaswiecicka What do you think of updating the label for the mobile alignment? Or maybe it's clearer to keep it 🤔 

Original comment

https://github.com/Shopify/dawn/pull/916#discussion_r757051602

Reply:
Aligned it shouldn't say mobile under the "mobile" header.